CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 5.39.0 (Unreleased)
+## 5.39.1 (Unreleased)
+
+BUG FIXES:
+* datastream: removed a breaking change in `google_datastream_stream` that made one of `destination_config.0.bigquery_destination_config.0.merge` or `destination_config.0.bigquery_destination_config.0.append_only` required ([#7835](https://github.com/hashicorp/terraform-provider-google-beta/pull/7835))
```

Comment:
I'm not sure if we have a standard way of formatting this but it seems like it could be nice to include the version that introduced the breaking change? And maybe fixed instead of removed? 🤷  But really probably fine as-is.

```suggestion
* datastream: fixed a breaking change in 5.39.0 `google_datastream_stream` that made one of `destination_config.0.bigquery_destination_config.0.merge` or `destination_config.0.bigquery_destination_config.0.append_only` required ([#7835](https://github.com/hashicorp/terraform-provider-google-beta/pull/7835))
```